rasa/core/training/structures.py

```diff
@@ -715,6 +715,7 @@ def dfs(node):
 
         while unprocessed:
             dfs(unprocessed.pop())
+        removed_edges = sorted(removed_edges)
```

Comment:
@Ghostvv types check failing because sorted converting set() to list() within topological_sort. The error is "attribute-error no attribute add for list". can you suggest how to fix this?